docs/user-stories-prod.md

```diff
@@ -5,47 +5,18 @@ title: User Stories in Productie
 
```

Comment:
De master branch versie is de juiste. Overschrijving door de develop branch versie is echter gene probleem aangezien het mergen van deze PR de actions in werking zet waarmee dit bestand weer meteen wordt geupdate.

Reply:
klopt, maar toch handmatig aangepast door merge conflict